src/SourceBuild/content/eng/allowed-sb-binaries.txt

```diff
@@ -3,6 +3,11 @@
 # This file is used by the Binary Tool to remove binaries from the VMR
 # If importing a file, include the relative path to the file
 
+.git/
```

Comment:
Consider adding a comment indicating that this section is for well known non-checked in infrastructure paths created by prep with known binaries.